src/lib/journey/stages/stages.stories.js

```diff
@@ -80,80 +80,80 @@ RegistrationInteraction.play = async ({ canvasElement }) => {
   const news = canvas.getByRole('checkbox', { name: /news/ })
   const securityQuestion = canvas.getByLabelText('Select a security question')
   const securityAnswer = canvas.getByLabelText('Security Answer')
-  const toc = canvas.getByRole('checkbox', { name: 'Please accept our Terms and Conditions' });
+  const tocLink = canvas.getByRole('link', { name: 'View full Terms & Conditions' });
+  const toc = canvas.getByRole('checkbox', { name: 'Please accept our Terms & Conditions' });
```

Comment:
Fixing a failed test because of the small change in the TOC component from a previous PR.